src/app/story/storyrow/totaltimes/totaltimes.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="timeline-properties mobile-hide" (click)="sidebarOpen = !sidebarOpen; $event.preventDefault()">
```

Comment:
`timeline-properties` is not used as far as I can tell. Could you please remove it?